eng/helix/helix.proj

```diff
@@ -19,7 +19,7 @@
   <PropertyGroup Condition=" '$(CI)' == 'true' ">
     <HelixType>ci</HelixType>
     <Creator>aspnetcore</Creator>
-    <HelixBuild>$(BUILD_BUILDNUMBER)</HelixBuild>
+    <HelixBuild>$(BUILD_BUILDNUMBER).$(BUILD_BUILDID)</HelixBuild>
```

Comment:
Doesn't work since the BUILDID doesn't increment between re-runs. Will look into another mechanism.